### ISSUE TITLE
Blackmagic Hyperdeck Loop Feature

### ISSUE THREAD

**Issue by arnoldc28** [bitfocus/companion-module-bmd-hyperdeck#30]
Would it be possible to have a loop in and loop out feature added for the BMD hyperdeck studio mini. Looking in the hyperdeck manual it appears that mark in and mark out can be made but only with timecode and I have has a go but not able to find a way of getting 2 timecode points to store in a command at once (1 for the mark in and 1 for the mark out).

I have been using a hyperdeck with the latest firmware and also using Companion 2.1.2

Thanks and loving the work everyone is putting into this!

**Comment by sphlabs** [comment]
Yes, this should be possible. A new action could allow you to set in and out points (as timecode), with another action to clear the playrange.

**Comment by AndyHuska** [comment]
Link to my comment in what could be a related issue, or at least solved with a common solution:
https://github.com/bitfocus/companion-module-bmd-hyperdeck/issues/53#issuecomment-995203428

**Comment by sphlabs** [comment]
This is also related to #12

**Comment by LikeThisReview** [comment]
Set play range inpoint outpoint is supported in Hyperdeck module v3.0.0. Issue can be closed if you ask me
